docker/Dockerfile

```diff
@@ -61,6 +61,7 @@ FROM base as torch-tensorrt-builder-base
 ARG ARCH="x86_64"
 ARG TARGETARCH="amd64"
 
+RUN apt-get update
```

Comment:
`python3-setuptools` was not being found in certain cases from the `apt` registry